bindinfo/bind_record.go

```diff
@@ -234,6 +232,15 @@ func (br *BindRecord) isSame(other *BindRecord) bool {
 	return br.OriginalSQL == other.OriginalSQL
 }
 
+// size calculates the memory size of a BindRecord.
+func (br *BindRecord) size() float64 {
+	mem := float64(len(hack.Slice(br.OriginalSQL)) + len(hack.Slice(br.Db)))
+	for _, binding := range br.Bindings {
+		mem += binding.size()
```

Comment:
Though it is not changed in this PR, I find `(*Binding).size()` doesn't consider the size of `Binding.Hint` and `Binding.ID`.

Reply:
It seems is not easy to calculate the size of the `Binding.Hint`. I add the `Binding.ID`.